src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_assistive.scss

```diff
@@ -0,0 +1,29 @@
+/*doc
+---
+title: Screen reader only modifier
```

Comment:
IMHO this name is misleading. This is visually hidden, i.e. it is not hidden on a wide range of user agents apart from screen readers. This is a minor issue though.